### ISSUE TITLE
cpplint: no hint on names of constants

### ISSUE THREAD

**Issue by LoveDaisy** [google/styleguide#416]
The script cpplint.py seems not recognize constant. Say, if I write this line:
```c++
constexpr int SOME_NAME = 0;
```
the script will not give any hint. 

According to cppguide.html it is expected to give a warning on the name and suggest a name `kSomeName`.

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see https://github.com/google/styleguide/pull/837). please migrate to https://github.com/cpplint/cpplint now.
